lms/djangoapps/discussion/rest_api/tests/test_views.py

```diff
@@ -1360,10 +1360,7 @@ def setUp(self):
         super().setUp()
         self.url = reverse("thread-list")
 
-    @mock.patch(
```

Comment:
should we not keep these?

Reply:
Not needed as these tasks calls have been moved to handlers and they have their separate test cases.